packages/team-management/src/components/TeamsView/index.tsx

```diff
@@ -16,41 +15,105 @@ import {
   Organization,
   reducerName,
 } from '../../ducks/organizations';
-import { TEAMS_GET, TEAM_PRACTITIONERS, URL_ADD_TEAM } from '../../constants';
+import {
+  TEAMS_GET,
+  TEAM_PRACTITIONERS,
+  URL_ADD_TEAM,
+  ASSIGNED_LOCATIONS_AND_PLANS,
+} from '../../constants';
 import Table from './Table';
 import './TeamsView.css';
 import { Spin } from 'antd';
 import { Link } from 'react-router-dom';
 import { Practitioner } from '../../ducks/practitioners';
+import { RawAssignment } from '@opensrp/team-assignment';
```

Comment:
add `@opensrp/team-assignment` dependency to teams management package

Reply:
255c6f7d97893022b4afe71a0371df0fd17ab869